_docs/2017-01-01-index.md

```diff
@@ -151,10 +151,10 @@ which is a worthy read for anyone with a background in cryptocurrency
 theory. The core idea of Proof of Stake is that instead of wasting
 electricity on cracking computationally heavy problems, a node is
 selected to mint a new block, with a probability proportional
-to the amount of coins a particular node has. Every node of network can attempt to
-generate a block. If node has positive [stake](/cardano/proof-of-stake/#stake) it
-is called _a stakeholder_. If node eventually becomes chosen to mint block
-it becomes _slotleader_. You can read more about this process in
+to the amount of coins a particular node has. If node has positive
+[stake](/cardano/proof-of-stake/#stake) it is called _a stakeholder_.
+If node eventually becomes chosen to mint block
+it is called _a slotleader_. You can read more about this process in
```

Comment:
@gromakovsky I rewrote sentences in a different way. I didn't see definitions of _stakeholder_ and _slotleader_ earlier here, seems good place to introduce them. Also, we can't assume that people read paper before reading docs.